src/zarr/config.py

```diff
@@ -18,6 +18,6 @@
 
 def parse_indexing_order(data: Any) -> Literal["C", "F"]:
```

Comment:
noting that this function also exists in `zarr.common`... No need to fix here but we don't need both.